openreview/conference/templates/programchairWebfield.js

```diff
@@ -617,7 +617,7 @@ var calcReviewersComplete = function(reviewerMap, officialReviews) {
 
 var calcPaperReviewsComplete = function(noteMap, officialReviewMap) {
   return _.reduce(noteMap, function(numComplete, reviewerMap, n) {
-    var allSubmitted = officialReviewMap[n] && Object.values(reviewerMap).length === Object.values(officialReviewMap[n]).length;
+    var allSubmitted = officialReviewMap[n] && Object.values(reviewerMap).length >0 && Object.values(reviewerMap).length === Object.values(officialReviewMap[n]).length;
```

Comment:
Could we move `Object.values(reviewerMap).length` into a separate var so it's not calling Object.values twice?

Reply:
Done.